pkg/util/util.go

```diff
@@ -36,12 +37,43 @@ import (
 const (
 	// MiB - MebiByte size
 	MiB = 1024 * 1024
+	GiB = MiB * 1024
 )
 
-// RoundUpToMiB rounds up given quantity upto chunks of MiB
-func RoundUpToMiB(size int64) int64 {
+// RoundOffVolSize rounds up given quantity upto chunks of MiB/GiB
+func RoundOffVolSize(size int64) int64 {
 	requestBytes := size
-	return roundUpSize(requestBytes, MiB)
+	if requestBytes < GiB {
+		return roundUpSize(requestBytes, MiB)
+	}
+	size = roundUpSize(requestBytes, GiB)
+	// convert size back to MiB for rbd CLI
+	return size * GiB / MiB
+}
+
+func roundUpSize(volumeSizeBytes, allocationUnitBytes int64) int64 {
+	roundedUp := volumeSizeBytes / allocationUnitBytes
+	if volumeSizeBytes%allocationUnitBytes > 0 {
+		roundedUp++
+	}
+	return roundedUp
+}
+
+// RoundOffBytes converts roundoff the size
+// 1.1Mib will be round off to 2Mib same for GiB
+// size less than will be round off to 1MiB
```

Comment:
nit: size less than 1MiB will be round off to 1MiB.

Reply:
done